packages/manager/apps/telecom/src/app/telecom/pack/orderFollowUp/orderFollowUp.controller.js

```diff
@@ -0,0 +1,83 @@
+import isEmpty from 'lodash/isEmpty';
+import last from 'lodash/last';
+
+export default class OrderFollowUpCtrl {
+  /* @ngInject */
+  constructor(
+    $stateParams,
+    $translate,
+    $http,
+    TucToast,
+    TucToastError,
+    ORDER_STATUS,
+  ) {
+    this.$http = $http;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.TucToast = TucToast;
+    this.TucToastError = TucToastError;
+    this.ORDER_STATUS = ORDER_STATUS;
+  }
+
+  getByPackName(packName) {
+    const url = `/pack/xdsl/${packName}/orderFollowUp`;
+
+    return this.$http
+      .get(url)
+      .then((response) => {
+        return response.data;
+      })
+      .catch((err) => {
+        throw new this.TucToastError(err);
+      });
+  }
+
+  loadData() {
+    this.getByPackName(this.packName)
+      .then((data) => {
+        this.events = data.map((elt) => {
+          this.allSuccess = data.every((item) => item.status === 'done');
+
+          const comments = elt.comments
+            ? elt.comments
+                .map((thisComment) => `<span>${thisComment}</span>`)
```

Comment:
```suggestion
                .map((thisComment) => thisComment)
```
It isn't best practice to have html code in controller.